src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Updated [Http Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/http.md).
+  This library can emit either old, new, or both attributes. Users can control
```

Comment:
I think we need more elaborate changelog. old and new may not be very clear to users. @alanwest for help 😄 
Can be done as separate PR.

Reply:
I copied this statement from the other libraries. 
Need to make sure all other changelogs get updated as well.